.github/scripts/rosetta/setup.sh

```diff
@@ -7,12 +7,11 @@ echo "Install binaries"
 cargo install --locked --bin iota --path crates/iota
 cargo install --locked --bin iota-rosetta --path crates/iota-rosetta
 
-echo "create dedicated config dir for IOTA genesis"
-CONFIG_DIR="~/.iota/rosetta_config"
-mkdir -p $CONFIG_DIR
-
 echo "run IOTA genesis"
-iota genesis -f --working-dir $CONFIG_DIR
+CONFIG_DIR=~/.iota/iota_config
+if ! [ -d "$CONFIG_DIR" ]; then
+    iota genesis
```

Comment:
Would it work after a CI runner reset?

Reply:
It will yes. The directory then won't exist and `iota genesis` will run.

Reply:
Thanks!